### ISSUE TITLE
[13] Add Military feed to Dashboard

### ISSUE THREAD

**Issue by joebayles** [ArcGIS/air-c2-cop-python#13]
User Story
- As an operations officer I have the ability to see all current and planned military flights in the area, which allows me to better understand the current air operations situation.

Acceptance Criteria
- A 2D layer displays all aircraft currently in the air or planning on departing in a 24-hour period in Operations Dashboard.
- A list and a detail widget linked to the 2D layer in Operations Dashboard.

http://niris.nc3a.nato.int/

Estimation Notes:
Basically FlightAware for military flights
